qa/rpc-tests/bumpfee.py

```diff
@@ -58,6 +58,7 @@ def run_test(self):
 
         print("Running tests")
         dest_address = peer_node.getnewaddress()
+        test_bumpfee_metadata(rbf_node, dest_address)
```

Comment:
Quick explanatory note on this test move; the outputs picked by the node depend on the test ordering (ugh), and at the later stages the output being picked consistently did not have enough change for a fee bump to succeed.